tfhe/src/shortint/server_key/mod.rs

```diff
@@ -1367,6 +1379,43 @@ impl Ciphertext {
 }
 
 impl ServerKey {
+    pub(crate) fn get_best_cleaning(
```

Comment:
not sure about that name even if it's pub(crate), maybe 

get_min_pbs_for_bivariate_evaluation ? not sure it it's used for other things that the bivar PBS

a docstring would be welcome as well

Reply:
This function is used for `smart_add` and `smart_sub` only

Let me know if you have a better idea for the name

I'll add a docstring

Reply:
Renamed `binary_smart_op_optimal_cleaning_strategy`
Added a docstring